.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - "6.2"
+
+before_script:
+  - npm install -g gulp bower
+  - bower install
```

Comment:
I think this should have:

`script: gulp`
as per: https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Using-Gulp

From the same page the default `script` is `npm test` and I'm unclear as to whether `npm test` incorporates both the tests and the code style, both of which are covered by `gulp` or `gulp qa`